tests/test_cl2hpp.cpp

```diff
@@ -616,12 +611,26 @@ static void CL_CALLBACK test_program_release_callback(
 {
 }
 
+static cl_int clSetProgramReleaseCallback_set(
```

Comment:
I'm guessing you didn't intend these changes to be part of the PR.

Reply:
No, this was intended.  I can move this change into a separate PR if desired, but this change is needed to fix the tests on 32-bit Windows due to the way `CL_CALLBACK` is defined and handled by these unit tests.

Reply:
It would have been better as a separate PR indeed but, given it's fairly small/straightforward, I'm fine with accepting the PR as is.